pennylane/compiler/qjit_api.py

```diff
@@ -19,65 +19,82 @@
 
 
 def qjit(fn=None, *args, compiler="catalyst", **kwargs):  # pylint:disable=keyword-arg-before-vararg
-    """A decorator for just-in-time compilation of hybrid quantum programs in PennyLane.
+    """A just-in-time decorator for PennyLane and JAX programs using Catalyst.
 
     This decorator enables both just-in-time and ahead-of-time compilation,
-    depending on the compiler package and whether function argument type hints
-    are provided.
+    depending on whether function argument type hints are provided.
 
     .. note::
 
-        Currently, only two compilers are supported; the :doc:`Catalyst <catalyst:index>` hybrid
-        quantum-classical compiler, which works with the JAX interface, and CUDA Quantum.
-
-        For more details on Catalyst, see the :doc:`Catalyst documentation <catalyst:index>` and
-        :func:`catalyst.qjit`.
-
-    .. note::
-
-        Catalyst only supports the JAX interface and selected devices.
-        Supported backend devices for Catalyst include
-        ``lightning.qubit``, ``lightning.kokkos``, ``lightning.gpu``, and ``braket.aws.qubit``,
-        but **not** ``default.qubit``.
-
-        For a full list of supported devices, please see :doc:`catalyst:dev/devices`.
-
-        CUDA Quantum supports ``softwareq.qpp``, ``nvidia.custatevec``, and ``nvidia.cutensornet``.
+        Not all PennyLane devices currently work with Catalyst. Supported backend devices include
+        ``lightning.qubit``, ``lightning.kokkos``, ``lightning.gpu``, and ``braket.aws.qubit``. For
```

Comment:
Qrack might deserve a mention here.

Reply:
Probably better to include a link to the supported devices page 🤔 https://docs.pennylane.ai/projects/catalyst/en/stable/dev/devices.html